### PR TITLE
Fix query handling and example format in response to Notion changes

### PR DESCRIPTION
**Pull request by jamalex**
On Jan 9, Notion made changes to the API format for queries. This PR allows the new query format to be used, but manually constructed queries will need to be updated. See https://github.com/jamalex/notion-py/issues/94.

### CODE REVIEW

**Comment by kevinjalbert** [comment]
👏 Awesome work @jamalex.

I tried out the branch for #96 (which includes this PR) and this does indeed fix the issue with specifying filters to match that of the new format.

I did notice that my `default_query()` for a `collection_view` was not updating when I made changes. 🤔 I decided to make a new view and give that a shot, then I was hit with:

```
  File "/usr/local/lib/python3.7/site-packages/notion/collection.py", line 223, in default_query
    return self.build_query(**self.get("query", {}))
  File "/usr/local/lib/python3.7/site-packages/notion/collection.py", line 219, in build_query
    collection=self.collection, collection_view=self, **kwargs
TypeError: __init__() got an unexpected keyword argument 'filter_operator'
```

I dug around and started inspecting the returned structure from Notion:

```diff
    def default_query(self):
+       print(self.get().keys())
        return self.build_query(**self.get("query", {}))
```

And saw the following surprise:

```
dict_keys(['id', 'version', 'type', 'name', 'format', 'parent_id', 'parent_table', 'alive', 'query2'])
```

Then I checked what I got with my original `collection_view`:

```
dict_keys(['id', 'version', 'type', 'name', 'query', 'format', 'parent_id', 'parent_table', 'alive', 'page_sort', 'query2'])
```

`query2` 😮

I applied the following change and things started working as expected:

```diff
    def default_query(self):
-       return self.build_query(**self.get("query", {}))
+       return self.build_query(**self.get("query2", {}))
```

**TL;DR** It seems with their API change, they phased out `query` as a field on the internal structure of the `collection_view` (for older views -- although they still exist), and now `query2` is the field that is updated for the `default_query`.

**Comment by nachtwerk** [comment]
@jamalex as said Notion added some more changes it seems to the API. I tried going do the road @kevinjalbert suggested, but that openend a can of worms with `aggregate` and `aggregations` keywords that are mutually excluded (but by default, are both passed?)
Is there any help that you need?

**Comment by jamalex** [comment]
It looks as if they renamed "query2" back to "query", so this works again. :) I've merged and and will continue doing further testing (and merging of subsequent PRs) on it before releasing.
